core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,106 +1,112 @@
 <div class="oppia-static-header">
-  <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
+  <h1 [innerHTML]="'I18N_ABOUT_PAGE_HEADING' | translate"></h1>
 </div>
 <background-banner></background-banner>
 <div>
+<!--   <h1> {{ TAB_ID_ABOUT }} </h1>
+  <h1> {{ activeTabName }} </h1>
+  <h1> {{ TAB_ID_CREDITS }} </h1>
+  <h1> {{ aboutPageMascotImgUrl }} </h1>
+  <h1> {{ TAB_ID_FOUNDATION }} </h1>
+  <h1> {{ I18N_ABOUT_PAGE_TABS_FOUNDATION }} </h1> -->
   <div class="oppia-static-content-below-banner">
     <div class="oppia-page-card oppia-static-content">
       <div class="about-nav">
         <ul class="oppia-about-tabs">
-          <li ng-class="{'oppia-about-tabs-active': $ctrl.activeTabName === $ctrl.TAB_ID_ABOUT}">
-            <a ng-click="$ctrl.onTabClick($ctrl.TAB_ID_ABOUT)" href="" class="oppia-about-tabs-text" id="about" translate="I18N_ABOUT_PAGE_TABS_ABOUT"></a>
+          <li [ngClass]="{'oppia-about-tabs-active': activeTabName === TAB_ID_ABOUT}">
+            <a (click)="onTabClick(TAB_ID_ABOUT)" class="oppia-about-tabs-text" id="about" [innerHTML]="'I18N_ABOUT_PAGE_TABS_ABOUT' | translate"></a>
```

Comment:
Starting a thread for discussing the use of innerHTML in the context of security. 
@seanlip, @bansalnitish, @vojtechjelinek ptal!
Angular sanitized the HTML for us. My source for this information is the angular guide.  https://angular.io/guide/security#best-practices has more information on how angular sanitizes the HTML we are adding.

Reply:
Thanks for the pointer! Two questions:

- Can you point to an authoritative doc that explains "use innerhtml in a translation context" or that talks about translation of html content in general?
- Did you test this manually by adding script tags, etc.? What about oppia-noninteractive-image and similar tags?

Reply:
@seanlip
> Can you point to an authoritative doc that explains "use innerhtml in a translation context" or that talks about translation of html content in general?

https://github.com/oppia/oppia/pull/9272/#issue-415517123

I have mentioned in the pr description about the various ways in which translate can be used. 
> https://angular.io/guide/security#best-practices has more information on how angular sanitizes the HTML we are adding.

The guide here clearly states that it doesn't allow HTML in interpolation, i.e., `{{...}}`. It will convert all `<` to `&lt;` and other characters too. One can check here (https://angular.io/guide/security#sanitization-example). So that leaves us with the only choice in angular to interpolate HTML content with innerHTML.

This `[innerHTML]="'I18N_TAG' | translate"` combines angular features in this order:

1.  First, the i18n tag has the respected values translated in the language selected. (https://github.com/ngx-translate/core#6-use-html-tags)
2. Second, the translated value is then rendered using the innerHTML through property binding, where the angular sanitizes the content we provide. (https://angular.io/guide/security#sanitization-example).

> Did you test this manually by adding script tags, etc.? What about oppia-noninteractive-image and similar tags?

I didn't check it with oppia-noninteractive-image but I checked it with other tags capable of running js (like img onerror, etc). Is oppia-noninteractive-image capable of running js?

Reply:
> Can you point to an authoritative doc that explains "use innerhtml in a translation context" or that talks about translation of html content in general?

What I mean is, what do other projects do? I understand the current reasoning but it would also be helpful to see how others solve this problem in practice.

> I didn't check it with oppia-noninteractive-image but I checked it with other tags capable of running js (like img onerror, etc). Is oppia-noninteractive-image capable of running js?

Typically not, but I just wanted to check whether the sanitizer accepts custom tags or not.

Reply:
>  does ngx-translate itself have opinions on a best practice?

https://github.com/ngx-translate/core#6-use-html-tags

> how do other angular projects do i18n of html strings

From what I have seen, They use ngx-translate or they go ahead and implement angular's i18n. The three file formats supported in angular's i18n are:
1. XLIFF 1.2 (default)
2. XLIFF 2
3. XML Message Bundle (XMB) 

Here is an example:
HTML
```
<h1 i18n="User welcome|An introduction header for this sample@@introductionHeader">
  Hello i18n!
</h1>
```
messages.fr.xlf
```
      <trans-unit id="introductionHeader" datatype="html">
        <source>Hello i18n!</source>
        <target>Bonjour i18n !</target>
        <note priority="1" from="description">An introduction header for this sample</note>
        <note priority="1" from="meaning">User welcome</note>
      </trans-unit>
```
Description and meaning are just for translators to better understand the context, they can be ignored.

Seeing this would require a significant refactor and also the angular.json file, this was dropped.

Reply:
> What about oppia-noninteractive-image and similar tags?
I didn't find any oppia-noninteractive-image/link/etc in translations. Anyways it will be removed. By default angular treats everything as unsafe. In case they ever end up in the translations we will have to manually sanitize `oppia-noninteractive-image`

Reply:
OK, thanks! Then I would say, what you have is fine. Thank you for all the explanations.

Please address all remaining open comments and feel free to send to reviewers once you have completed the PR.